.env.example

```
@@ -18,7 +18,25 @@ PORT=2567
 DISCORD_CLIENT_ID=your_discord_client_id
 DISCORD_CLIENT_SECRET=your_discord_client_secret
 
+# ── Google sign-in (server verifies the ID token) ─────────────────────────────
+# OAuth 2.0 "Web application" client ID from Google Cloud Console.
+GOOGLE_CLIENT_ID=your_google_web_client_id.apps.googleusercontent.com
+
+# ── Apple sign-in (server verifies the identity token) ────────────────────────
+# Services ID = web sign-in audience; Bundle ID = native iOS app audience.
+APPLE_SERVICES_ID=com.yourcompany.durak.web
+APPLE_BUNDLE_ID=com.yourcompany.durak
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The APPLE_BUNDLE_ID key should go before the APPLE_SERVICES_ID key

(UnorderedKey)